src/fontstash.h

```diff
@@ -1041,6 +1058,18 @@ static FONSglyph* fons__getGlyph(FONScontext* stash, FONSfont* font, unsigned in
 	// Could not find glyph, create it.
 	scale = fons__tt_getPixelHeightScale(&font->font, size);
 	g = fons__tt_getGlyphIndex(&font->font, codepoint);
+	// Try to find the glyph in fallback fonts.
+	if (g == 0)
+	{
+		for (i = 0; i < font->nfallbacks; ++i)
+		{
+			fallbackGlyph = fons__getGlyph(stash, stash->fonts[font->fallbacks[i]], codepoint, isize, iblur);
+			if (fallbackGlyph != NULL)
```

Comment:
The original implementation checks if `glyph->index != 0`. However, this only works for `stb_truetype`. To make it work on both `stb_truetype` and `freetype`, we can only check if the fallback glyph is NULL. Checking `glyph->index` causes issue when using `freetype` backend.